operator/api/v1alpha1/keptntask_types.go

```diff
@@ -80,3 +85,44 @@ type KeptnTaskList struct {
 func init() {
 	SchemeBuilder.Register(&KeptnTask{}, &KeptnTaskList{})
 }
+
+func (i *KeptnTask) SetStartTime() {
+	if i.Status.StartTime.IsZero() {
+		i.Status.StartTime = metav1.NewTime(time.Now().UTC())
+	}
+}
+
+func (i *KeptnTask) SetEndTime() {
+	if i.Status.EndTime.IsZero() {
+		i.Status.EndTime = metav1.NewTime(time.Now().UTC())
+	}
+}
+
+func (i *KeptnTask) IsStartTimeSet() bool {
+	return !i.Status.StartTime.IsZero()
+}
+
+func (i *KeptnTask) IsEndTimeSet() bool {
+	return !i.Status.EndTime.IsZero()
+}
+
+func (i KeptnTask) GetActiveMetricsAttributes() []attribute.KeyValue {
+	return []attribute.KeyValue{
+		attribute.Key("KeptnApp").String(i.Spec.AppName),
```

Comment:
I'd suggest keeping it consistent with traces attribute and use `keptn.deployment.app_name`. 
This can be done after #115 is merged and contains predefined keys:
https://github.com/keptn-sandbox/lifecycle-controller/pull/115/files#diff-9c892c3e1966aa21e9980924b7b885fd2aed3d453b79e321569b2e7eadf99fe7R11

Reply:
sure, we can wait form merging #115 and afterwards adapt this